src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/TaintedDataAnalysis/TaintedDataAnalysis.TaintedDataOperationVisitor.cs

```diff
@@ -228,8 +230,37 @@ public override TaintedDataAbstractValue VisitObjectCreation(IObjectCreationOper
                 {
                     return TaintedDataAbstractValue.NotTainted;
                 }
-                else if (this.DataFlowAnalysisContext.SourceInfos.IsSourceMethod(method))
+                else if (this.DataFlowAnalysisContext.SourceInfos.IsSourceMethod(
+                    method,
+                    out PooledHashSet<IsInvocationTaintedWithPointsToAnalysis> evaluateWithPointsToAnalysis,
+                    out PooledHashSet<IsInvocationTaintedWithValueContentAnalysis> evaluateWithValueContentAnalysis))
                 {
```

Comment:
Oh I forgot to mention, using PooledHashSet means you need to .Free() yourself.  So there should be a try {} finally {} inside this if block.